mypy.ini

```diff
@@ -0,0 +1,3 @@
+[mypy]
+files=wikipron,data,tests
+ignore_missing_imports=true
```

Comment:
Ah, now I see why you've been omitting the args when calling the `mypy` command. This repo doesn't have a `setup.cfg` or any tool-specific `.ini` for this sort of configuration, and I think we'd like to keep things this way. Is it possible to remove this `mypy.ini` file and put these arguments together with the `mypy` command as suggested in previous comments?